plugins/woocommerce-blocks/assets/js/blocks/product-collection/types.ts

```diff
@@ -80,18 +83,49 @@ export interface ProductCollectionQuery {
 	priceRange?: undefined | PriceRange;
 }
 
+export type ProductCollectionEditComponentProps =
+	BlockEditProps< ProductCollectionAttributes > & {
+		openCollectionSelectionModal: () => void;
+	};
+
 export type TProductCollectionOrder = 'asc' | 'desc';
 export type TProductCollectionOrderBy =
 	| 'date'
 	| 'title'
 	| 'popularity'
 	| 'rating';
 
-export type DisplayLayoutToolbarProps = {
+export type DisplayLayoutControlProps = {
 	displayLayout: ProductCollectionDisplayLayout;
 	setAttributes: ( attrs: Partial< ProductCollectionAttributes > ) => void;
 };
 export type QueryControlProps = {
 	query: ProductCollectionQuery;
 	setQueryAttribute: ( attrs: Partial< ProductCollectionQuery > ) => void;
 };
+
+export enum CoreCollectionNames {
+	PRODUCT_CATALOG = 'woocommerce-blocks/product-collection/product-catalog',
+	CUSTOM = 'woocommerce-blocks/product-collection/custom',
+	BEST_SELLERS = 'woocommerce-blocks/product-collection/best-sellers',
+	FEATURED = 'woocommerce-blocks/product-collection/featured',
+	NEW_ARRIVALS = 'woocommerce-blocks/product-collection/new-arrivals',
+	ON_SALE = 'woocommerce-blocks/product-collection/on-sale',
+	TOP_RATED = 'woocommerce-blocks/product-collection/top-rated',
+}
```

Comment:
Nit: Given the merge into the monorepo, would it be more appropriate to refer to it as "woocommerce" instead of "woocommerce-blocks"?

Reply:
Good idea, fixed!